packages/protocol/scripts/bash/release-on-devchain.sh

```diff
@@ -39,8 +39,6 @@ mv build/contracts $BUILD_DIR
 echo "- Run local network"
 # yarn devchain generate-tar devchain.tar.gz
 yarn devchain run-tar devchain.tar.gz >> $LOG_FILE &
-GANACHE_PID=$!
```

Comment:
Can we get the PID here and print it out, in case the script fails and the user can kill the process manually?

Reply:
We'd have to give ourselves enough time so we're pretty sure ganache has bound the port. I guess right now we're assuming that `yarn build` below takes enough time to not need a `sleep`, so we could put `lsof` after that, but that would leave us with the edge case of not printing it out after a failed build.

I guess that could be acceptable (this first build is of an old branch, which should have been tested and building correctly for a while now).